Lib/annotationlib.py

```diff
@@ -97,16 +93,12 @@ def evaluate(self, *, globals=None, locals=None, type_params=None, owner=None):
 
         If the forward reference cannot be evaluated, raise an exception.
         """
-        if self.__forward_evaluated__:
-            return self.__forward_value__
         if self.__cell__ is not None:
             try:
                 value = self.__cell__.cell_contents
             except ValueError:
                 pass
             else:
-                self.__forward_evaluated__ = True
-                self.__forward_value__ = value
                 return value
```

Comment:
We can remove this `else` branch and just write: `try: return self.__cell__.cell_contents`

Reply:
Good point, sorry I missed this comment